book.rb

```diff
@@ -0,0 +1,14 @@
+class Book
+  attr_accessor :title, :author
+  attr_reader :rentals
+
```

Comment:
- [OPTIONAL] Great work so far 👍🏻 However the *Getters and Setters for @rentals should be present as well to enable the read and write options. You may update your code like this:

```ruby
  attr_accessor :title, :author, :rentals
```